python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -48,6 +48,10 @@ def from_pretrained(cls,
         if bigdl_transformers_int4:
             # Avoid KeyError
             kwargs["ignore_mismatched_sizes"] = True
+            # Speed up when loading model
+            kwargs["low_cpu_mem_usage"] = True
+            # Avoid reading from local file at the first initialization
+            kwargs["state_dict"] = {}
 
         model = cls.HF_Model.from_pretrained(*args, **kwargs)
         print("Note: If there are warnings about mismatched during the loading process, "
```

Comment:
"Note: If there are warnings during the model loading process, "
              "they can be safely ignored."
              "The model will be loaded with INT4 optimizations applied")

Reply:
fixed